apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -1,19 +1,19 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tutorial - Components</title>
-  <link
-    rel="canonical"
-    href="https://tuono.dev/documentation/tutorial/components"
-  />
-</Head>
+<MetaTags
+  title="Tuono - Components"
+  canonical="https://tuono.dev/documentation/tutorial/components"
+  description="Learn how to manage the components in a Tuono codebase"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Tutorial" href="/documentation/tutorial" />
-  <Element label="Components" />
-</Breadcrumbs>
+<Breadcrumbs
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a word (ENGLISH_WORD_REPEAT_RULE)
Suggestions: `Breadcrumbs`
Rule: https://community.languagetool.org/rule/show/ENGLISH_WORD_REPEAT_RULE?lang=en-US
Category: MISC
<!-- __reviewdog__:ChAyYWI5NzJjMTA0MDJlNDk0EgxMYW5ndWFnZVRvb2w= -->